includes/class-wc-stripe-customer.php

```diff
@@ -120,7 +120,10 @@ protected function generate_customer_request( $args = array() ) {
 			// translators: %1$s First name, %2$s Second name, %3$s Username.
 			$description = sprintf( __( 'Name: %1$s %2$s, Username: %s', 'woocommerce-gateway-stripe' ), $billing_first_name, $billing_last_name, $user->user_login );
 
+			$billing_full_name = "$billing_first_name $billing_last_name";
```

Comment:
I like this but it might be possible (unlikely but possible) for one of these names to be empty. I recommend 1) using the string concatenation operator to combine the strings, 2) trim-ming the result and 3) only adding it to the array if not-empty here and for the non-user case below